### PR TITLE
Add instructions for downloading oc and using hosted template

### PR DESCRIPTION
**Pull request by sspeiche**
ptal @bparees

### CODE REVIEW

File: examples/jenkins/pipeline/README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ utilize pods run on OpenShift as Jenkins slaves.
 
 To walk through the example:
 
+0. If using `oc cluster up`, be sure to grab the [1.3alpha1](https://github.com/openshift/origin/releases/tag/v1.3.0-alpha.1)
```
Comment (bparees) [inline examples/jenkins/pipeline/README.md]: should we just point them to latest instead of a specific version?

**Comment by sspeiche** [comment]
Make the link change, ptal again @bparees

**Comment by bparees** [comment]
lgtm [merge]

**Comment by bparees** [comment]
[merge]


On Fri, Jun 10, 2016 at 8:05 PM, OpenShift Bot <notifications@github.com>
wrote:

> continuous-integration/openshift-jenkins/merge ABORTED (
> https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4726/)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9263#issuecomment-225323059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEvl3vqQUjFhQRHA5H_aHi8h8sGAk6Rpks5qKfvGgaJpZM4IybPa>
> .
>



-- 
Ben Parees | OpenShift

**Comment by bparees** [comment]
flake https://github.com/openshift/origin/issues/9285
[merge]


On Fri, Jun 10, 2016 at 9:25 PM, OpenShift Bot <notifications@github.com>
wrote:

> continuous-integration/openshift-jenkins/merge FAILURE (
> https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4734/)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9263#issuecomment-225328900>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEvl3uBxmPpJ5wtudt0EzRIoBKiiCXMlks5qKg6HgaJpZM4IybPa>
> .
>



-- 
Ben Parees | OpenShift

**Comment by bparees** [comment]
this is the pr that spawned a thousand flakes...

flake https://github.com/openshift/origin/issues/9286

[merge]


On Fri, Jun 10, 2016 at 10:45 PM, OpenShift Bot <notifications@github.com>
wrote:

> continuous-integration/openshift-jenkins/merge FAILURE (
> https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4735/)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9263#issuecomment-225332660>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEvl3j7pRGjX8jqSOYPDy9HWL7u6ybk6ks5qKiFLgaJpZM4IybPa>
> .
>



-- 
Ben Parees | OpenShift

**Comment by liggitt** [comment]
Are we sure the doc file change in this PR isn't causing the flakes?

**Comment by bparees** [comment]
@liggitt it's bad enough that i actually just double checked what this PR is actually changing :)

**Comment by bparees** [comment]
and it's about to flake again, this time on /data/src/github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/empty_dir.go

**Comment by bparees** [comment]
[merge]

**Pull request merged**